swagger/Budget.yaml

```diff
@@ -0,0 +1,200 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Budget API
+  description: |
+    REST API service for providing/storing budget data by request
+schemes:
+- http
+basePath: /api/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /spendings:
+    get:
+      summary: Get a list of spendings
+      description: ""
+      tags:
+      - Spendings
+      parameters:
+      - name: category
+        in: query
+        description: budget category
+        type: string
+        required: false
+        enum:
+        - entertainment
+        - car
+        - bill
+        - food
+        - home
+        - education
+        - other
+      - name: label
```

Comment:
`labels`. User will be able to filter by several labels.
E.g.: `/api/v1/spangins?labels=label1,label2`